### PR TITLE
disambiguate these two docs.

### PR DESCRIPTION
**Pull request by xiaket**
Hi team,

I would like to add two sentences into these two pages so as to make it clear that we have two engines, one secret engine and one database secret engine, as they had confused me earlier.

Thanks!

### CODE REVIEW

File: website/content/docs/secrets/mongodbatlas.mdx [hunk 1]
```diff
@@ -16,6 +16,8 @@ project or organization with appropriate role(s). The MongoDB Atlas Programmatic
 Private Keys are returned to the caller. To learn more about Programmatic API Keys visit the
 [Programmatic API Keys Doc](https://docs.atlas.mongodb.com/reference/api-docs/apiKeys/).
 
+Please do not confuse this with MongoDB Atlas Database secrets engine described [here](/vault/docs/secrets/databases/mongodbatlas).
```
Comment (maxb) [inline website/content/docs/secrets/mongodbatlas.mdx:19]: The thing you are linking to here is not a secrets engine - it is a database plugin

```suggestion
Please do not confuse this with MongoDB Atlas database plugin described [here](/vault/docs/secrets/databases/mongodbatlas).
```

Comment (schavis) [inline website/content/docs/secrets/mongodbatlas.mdx:19]: <Note>
  The information below relates to the MongoDB Altas <b>secrets engine</b>. Refer to the
  <a href="vault/docs/secrets/databases/mongodbatlas">MongoDB Atlas database secrets engine</a>
  for information about using the MongoDB Atlas database plugin for the Vault
  database secrets engine.
</Note>

Comment (schavis) [inline website/content/docs/secrets/mongodbatlas.mdx:19]:
```suggestion
<Note>
  The information below relates to the MongoDB Altas <b>secrets engine</b>. Refer to the
  <a href="vault/docs/secrets/databases/mongodbatlas">MongoDB Atlas database secrets engine</a>
  for information about using the MongoDB Atlas database plugin for the Vault
  database secrets engine.
</Note>
```
style corrections: avoid "please".

**Review by schavis — CHANGES_REQUESTED** [review]

File: website/content/docs/secrets/databases/mongodbatlas.mdx [hunk 2]
```diff
@@ -17,6 +17,8 @@ it uses a public and private key pair to authenticate.
 See the [database secrets engine](/vault/docs/secrets/databases) docs for
 more information about setting up the database secrets engine.
 
+Please do not confuse this with MongoDB Atlas secrets engine described [here](/vault/docs/secrets/mongodbatlas).
```
Comment (VioletHynes) [inline website/content/docs/secrets/databases/mongodbatlas.mdx:20]: <Note>
  The information below relates to the MongoDB Altas <b>database plugin</b> for the Vault database secrets engine.
  Refer to the <a href="vault/docs/secrets/mongodbatlas">MongoDB Atlas secrets engine</a>
  for information about using the MongoDB Atlas secrets engine for the Vault.
</Note>

File: website/content/docs/secrets/databases/mongodbatlas.mdx [hunk 3]
```diff
@@ -17,6 +17,8 @@ it uses a public and private key pair to authenticate.
...
+
```
Comment (VioletHynes) [inline website/content/docs/secrets/databases/mongodbatlas.mdx:21]:
```suggestion
<Note>
  The information below relates to the MongoDB Altas <b>database plugin</b> for the Vault database secrets engine.
  Refer to the <a href="vault/docs/secrets/mongodbatlas">MongoDB Atlas secrets engine</a>
  for information about using the MongoDB Atlas secrets engine for the Vault.
</Note>
```

**Comment by VioletHynes** [comment]
Thanks for the suggestion! I've updated some of the docs based on the suggestions here and I'll look to merge this PR.

File: website/content/docs/secrets/mongodbatlas.mdx (hunk 1 above)
Comment (VioletHynes) [reply website/content/docs/secrets/mongodbatlas.mdx]: Merged the suggestion. Thanks!

**Review by VioletHynes — APPROVED** [review]

**Pull request merged**